code/components/citizen-server-impl/include/NoticeLogicProcessor.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <StdInc.h>
+#include <ServerInstanceBase.h>
+#include <ResourceManager.h>
+
+#include <json.hpp>
+
+using json = nlohmann::json;
```

Comment:
better put this one in the class itself, also better for consumers of this processor, hence: `NoticeLogicProcessor::json`

Reply:
I'm not entirely sure I understand, if I move the `#include <json.hpp>` from the processor's .h to the .cpp I lose the ability to declare function prototypes with JSON types in the argument list. There's no separate `json` class that NoticeLogicProcessor declares or implements, I just stick to `using json = nlohmann:json;` because it makes referencing the 3rd party (nlohmann) types less lengthy and it's what [ServerNucleus.cpp did too before](https://github.com/citizenfx/fivem/blob/master/code/components/citizen-server-impl/src/ServerNucleus.cpp#L25) this change.

Reply:
Alright, next time you can just put the `using ...` or `typedef ...` in the class/struct itself. Look at `std` types for a reference